python/ray/serve/_private/default_impl.py

```diff
@@ -67,16 +69,55 @@ def create_init_handle_options(**kwargs):
     return _InitHandleOptions.create(**kwargs)
 
 
+_global_async_loop = None
```

Comment:
Should this be renamed to `_global_async_router_loop` or something similar, to indicate it's the loop for the router?

Reply:
yea good idea